api/services/handle_event_service.py

```diff
@@ -60,6 +61,7 @@ def handle_event_service(self, form_data):
 
             call_log_event.CallLogEventService.create_call_log_event(self, data)
             if data["call_status"] == "Missed":
+                user.UserService.create_user(self, data)

```

Comment:
The function create_user() should create a user and not check if user exists or not. Lets rename it to handle_user_creation()

Reply:
updated